app/main/posts/views/post-card.directive.js

```diff
@@ -47,6 +47,9 @@ function PostCardDirective(FormEndpoint, PostLockService, $rootScope) {
                     // But ignore the action
                     return;
                 }
+                if (evt && evt.key && (evt.key !== ' ' && evt.key !== 'Enter' && evt.key !== 'Spacebar')) {
```

Comment:
@Angamanga  @rowasc, the ng-keypress attribute was applicable to all keys so I wrote this conditional statement so as to limit its accessibility to only enter or spacebar key